src/cli.js

```diff
@@ -8,6 +8,9 @@ import {Program} from './program';
 
 export function main() {
   let program = new Program();
+  // yargs uses magic camel case expansion to expose options on the
+  // final argv object. For example, the 'build-dir' option is available
+  // as argv.buildDir.
```

Comment:
This approach can't catch a typo in an option configuration or a removal of an option that some handler relies on. I can address that in a follow-up patch if I can figure out how to do it :) 

This patch is still nice because *all* handler's adhere to a contract for what options they receive. If we had ten handlers, for example, each one of their function signatures would be validated.